core/test_helpers.py

```diff
@@ -67,7 +67,30 @@ def create_test_interactive_user(
             k: v for k, v in custom_props.items() if hasattr(InteractiveUser, k)
         }
     if roles is None:
-        roles = [7, 1, 2, 3, 4, 5, 6]
+        # Create a test role with default permissions instead of hardcoded role IDs
+        default_perm_names = [
+            "gql_query_roles_perms",
+            "gql_mutation_create_roles_perms",
+            "gql_mutation_update_roles_perms",
+            "gql_mutation_replace_roles_perms",
+            "gql_mutation_duplicate_roles_perms",
+            "gql_mutation_delete_roles_perms",
+            "gql_query_users_perms",
+            "gql_query_users_profile_perms",
+            "gql_mutation_create_users_perms",
+            "gql_mutation_update_users_perms",
+            "gql_mutation_delete_users_perms",
+            "gql_query_enrolment_officers_perms",
+            "gql_mutation_create_enrolment_officers_perms",
+            "gql_mutation_update_enrolment_officers_perms",
+            "gql_mutation_delete_enrolment_officers_perms",
+            "gql_query_claim_administrator_perms",
+            "gql_mutation_create_claim_administrator_perms",
+            "gql_mutation_update_claim_administrator_perms",
+            "gql_mutation_delete_claim_administrator_perms",
+        ]
```

Comment:
Can all these perms and below perms be in fixture files so 1) the test file is slimmer and 2) those fixtures may potentially be reused, even just as sample setups?